services/portal/values-idfdev.yaml

```diff
@@ -4,7 +4,11 @@ resources:
   limits:
     memory: "2Gi"
 
+image:
+  tag: "suit-2022.3"
+
 config:
+  hipsUrl: "https://irsa.ipac.caltech.edu/data/hips/CDS/DSS2/color/"
```

Comment:
Why is DSS2/color used here (in values-idfdev.html)?  Is a local HiPS not available on data-dev?

Reply:
yes, it is not available.